src/lib/strings.js

```diff
@@ -0,0 +1,32 @@
+function sayHello (string) {
+  return "Hello, " + string + "!";
+};
```

Comment:
Why is there a space after the name of the functions and before the opening bracket in this file? These should all be removed. The return statement could also be changed to use a template literal as described above:
```
  return `Hello, ${string}!`;
```